app/assets/javascripts/application.js

```diff
@@ -48,4 +48,4 @@
 //= require graph.js
 //= require short-code-forms/dist/short-code-prompts.js
 //= require wikis.js
-//= require chart.js/dist/Chart.js
+//= require chart.js/dist/Chart.js
```

Comment:
Same here -- i think your editor may be making tweaks, but they don't seem relevant to the PR. Appreciated!